Common/Configurations/ConfigurationProvider.cs

```diff
@@ -23,7 +23,8 @@ public string GetConfigurationSettingValueOrDefault(string configurationSettingN
                 if (!this.configuration.ContainsKey(configurationSettingName))
                 {
                     string configValue = CloudConfigurationManager.GetSetting(configurationSettingName);
-                    bool isEmulated = Environment.CommandLine.Contains("iisexpress.exe") ||
+                    bool isEmulated = Environment.CommandLine.Contains("iisexpress.exe") || 
+                        Environment.CommandLine.Contains("w3wp.exe") ||
```

Comment:
I think Charlie was running RM inside w3wp worker process. You can double check with him to figure out why. But it should be harmless.